src/Microsoft.AspNetCore.Hosting/WebHostExtensions.cs

```diff
@@ -15,11 +16,61 @@ namespace Microsoft.AspNetCore.Hosting
     public static class WebHostExtensions
     {
         /// <summary>
+        /// Starts the host.
+        /// </summary>
+        /// <param name="host"></param>
+        /// <returns></returns>
+        public static void Start(this IWebHost host)
+        {
+            host.StartAsync().GetAwaiter().GetResult();
+        }
+
+        /// <summary>
+        /// Starts the host.
+        /// </summary>
+        /// <param name="host"></param>
+        /// <returns></returns>
+        public static Task StartAsync(this IWebHost host)
+        {
+            return host.StartAsync(CancellationToken.None);
+        }
+
+        /// <summary>
+        /// Gracefully stops the host.
+        /// </summary>
+        /// <param name="host"></param>
+        /// <returns></returns>
+        public static Task StopAsync(this IWebHost host)
+        {
+            return host.StopAsync(CancellationToken.None);
+        }
+
+        /// <summary>
+        /// Attempts to gracefully stop the host with the given timeout.
+        /// </summary>
+        /// <param name="host"></param>
+        /// <param name="timeout"></param>
+        /// <returns></returns>
+        public static Task StopAsync(this IWebHost host, TimeSpan timeout)
```

Comment:
It looks like this overrides `WebHostOptions.ShutdownTimeout`. Any tests verifying that?